SetupAppveyor.ps1

```diff
@@ -0,0 +1,15 @@
+# only run if on appveyor, don't want to disable strong name verification elsewhere
```

Comment:
why is this conditional needed? this entire script is specific to app veyor right? I.e. it will only be called on app veyor builds